modules/jaeger-install.adoc

```diff
@@ -36,12 +36,7 @@ Do not install Community versions of the Operators. Community Operators are not
 
 . Select the *stable* Update Channel.
```

Comment:
The `stable` channel is still the default, and will automatically upgrade Jaeger instances as new higher versions are released. There are also maintenance channels (e.g. `1.13-stable` and `1.17-stable` currently) that enable users to select a particular minor version they wish to remain on, and receive bug and security patches (while the version is still within the support window - not sure how best to phrase that?).